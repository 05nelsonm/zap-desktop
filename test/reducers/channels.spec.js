import channelsReducer, {
  SET_CHANNEL_FORM,
  SET_CHANNEL,
  GET_CHANNELS,
  RECEIVE_CHANNELS,
  OPENING_CHANNEL,
  OPENING_SUCCESSFUL,
  OPENING_FAILURE
} from '../../app/reducers/channels'

describe('reducers', () => {
  describe('channelsReducer', () => {
    it('should handle initial state', () => {
      expect(channelsReducer(undefined, {})).toMatchSnapshot()
    })
  })
})