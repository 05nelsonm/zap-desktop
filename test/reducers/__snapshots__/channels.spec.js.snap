// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reducers channelsReducer should handle initial state 1`] = `
Object {
  "channel": null,
  "channelForm": Object {
    "isOpen": false,
    "local_amt": "",
    "node_key": "",
    "push_amt": "",
  },
  "channels": Array [],
  "channelsLoading": false,
  "openingChannel": false,
  "pendingChannels": Object {
    "pending_closing_channels": Array [],
    "pending_force_closing_channels": Array [],
    "pending_open_channels": Array [],
    "total_limbo_balance": "",
  },
}
`;
